pandas/tests/indexing/test_indexing.py

```diff
@@ -131,6 +128,14 @@ def test_setitem_ndarray_3d(self, index, obj, idxr, idxr_id):
             err = ValueError
             msg = r"Buffer has wrong number of dimensions \(expected 1, got 3\)|"
 
+        if (
+            (len(indices) == 0)
+            and (idxr_id == "iloc")
+            and isinstance(obj, pd.DataFrame)
+        ):
+            # TODO: Seems to be bugged
+            pytest.xfail("This doesn't raise")
```

Comment:
Wasn't sure what to do here. Is this ok or should this be fixed? I added an `xfail` for now, but can create a follow-up issue

Reply:
can you split this to a separate test for now (and just xfail the entire test; we can come back later and fix), but rather not add this code. also create an issue (and reference it here).